diff --git a/src/ConfigPagesStorage.php b/src/ConfigPagesStorage.php
index b08771a..f67d298 100644
--- a/src/ConfigPagesStorage.php
+++ b/src/ConfigPagesStorage.php
@@ -21,7 +21,7 @@ public function load($id) {
 
     // Default behavior allow to load entity by ID.
     if (is_numeric($id)) {
-      $entities = $this->loadMultiple([$id]);
+      $entities = parent::loadMultiple([$id]);
       $entity = isset($entities[$id]) ? $entities[$id] : NULL;
     }
     else {
@@ -31,4 +31,17 @@ public function load($id) {
     return $entity;
   }
 
+  public function loadMultiple(array $ids = NULL) {
+    $entities = [];
+
+    foreach ($ids as $id){
+      $entity = $this->load($id);
+      if (!empty($entity)) {
+        $entities[$entity->id()] = $entity;
+      }
+    }
+
+    return !empty($entities) ? $entities : NULL;
+  }
+
 }
